pkg/httpmock/stub.go

```diff
@@ -57,6 +57,33 @@ func GraphQL(q string) Matcher {
 	}
 }
 
+func MatchGraphQLMutation(q string, cb func(map[string]interface{}) bool) Matcher {
```

Comment:
I like this. I think naming it `GraphQLMutationMatcher` is slightly better although we don't have a set naming scheme for matchers.